src/fontra/backends/ufo_utils.py

```diff
@@ -24,5 +24,5 @@ def extractGlyphNameAndUnicodes(
                 "actual file name does not match predicted file name: "
                 f"{refFileName} {fileName} {glyphName}"
             )
-    unicodes = [int(u, 16) for u in _unicodePat.findall(data)]
-    return glyphName, unicodes
+    codePoints = [int(u, 16) for u in _codePointPat.findall(data)]
```

Comment:
But: keep the variable name, only revert the `_codePointPat` name.

Reply:
OK. Did it.